### ISSUE TITLE
Review and update COVID-19 chatbot

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#30511]
## Issue Description
Now that we've made so many updates and have received new information about COVID-19 vaccines, we need to make sure the information we have on our chatbot is still correct and up to date and if there should be new questions added.

## Tasks
- [x] Review chatbot for any outdated content (Selina)
- [x] Review vaccine pages to determine if new questions should be added to chatbot (Selina)
- [x] Confirm with Danielle what should be added (Selina)
- [x] Update chatbot (Selina)
- [x] Review changes (Randi)

## Acceptance Criteria
- [x] Chatbot changes are live

**Comment by lunascoop** [comment]
Content updates for questions 31, 67, 68, 8, 63, 57, 56, 62, 12, 17, 22 (I’m not sure if I changed anything), 59, 41, 45, 58, 23, 24, 55, 25, 51, 52, 60, 20, 69, 71, 70, 84, 81, 85, 80, 75, 76, 77

Notes

17 (is a combination of How do I refill my prescriptions? And How can I get more refills for my prescriptions? On the FAQs page)
69 (Didn’t change question title)
Replaced 72 w/84
85 (Replaced 74 and moved to after q. 70) May also want to add “else to q. 85’s title to better distinguish it from q. 70)
80 (changed “know” to “do” in question title to match accordion)
Changed q. 75 title to match accordion
Updated q. 76 title
q. 83 is the vaccine records online page intro
Deleted questions 61( 61 is kinda combined w/62)16, 21, 73, 74 (answer included in q.85)
